### PR TITLE
change service-ca import to runtime for sonarcube and jenkins

### PR DESCRIPTION
**Pull request by oalyman**

### CODE REVIEW

File: .github/workflows/continuous-integration-workflow.yml [hunk 1]
```diff
@@ -15,6 +27,14 @@ jobs:
...
         run: |
           shellcheck sonarqube/*.sh
+      -
+        name: Build image
+        working-directory: sonarqube
+        run: |
+          docker build \
+          --build-arg sonarVersion=${{ matrix.version }} \
+          --build-arg sonarDistributionUrl="https://binaries.sonarsource.com/Distribution/sonarqube/sonarqube-${{ matrix.version }}.zip" \
+          -t sonarcube .
```
Comment (michaelsauter) [inline .github/workflows/continuous-integration-workflow.yml:37]: `test.sh` already builds the image, so this should not be necessary?

File: jenkins/master/Dockerfile [hunk 2]
```diff
@@ -36,6 +32,8 @@ RUN mkdir -p /etc/opendevstack \
     && echo "  \"sonarqubeVersion\": \"$SONAR_VERSION\"" >> /etc/opendevstack/config.json \
     && echo "}" >> /etc/opendevstack/config.json
 
+RUN chmod g+w $JAVA_HOME/lib/security/cacerts
```
Comment (michaelsauter) [inline jenkins/master/Dockerfile:35]: Were you able to verify that this works both RedHat and CentOS based? It looks right to me, but would be good to verify.

File: .github/workflows/continuous-integration-workflow.yml (hunk 1 above)
Comment (oalyman) [reply .github/workflows/continuous-integration-workflow.yml]: I remove the docker build for sonarcube

File: jenkins/master/Dockerfile (hunk 2 above)
Comment (oalyman) [reply jenkins/master/Dockerfile:35]: Yes, works.

**Pull request merged**